src/content/docs/en/guides/images.mdx

```diff
@@ -4,159 +4,673 @@ description: Learn how to use images in Astro.
+
+We recommend that local images are kept in `src/` when possible so that Astro can transform, optimize and bundle them. Files in the `/public` directory are always served or copied into the build folder as-is, with no processing. 
+
+Your local images stored in `src/` can be used by all files in your project: `.astro`, `.md`, `.mdx`, `.mdoc` and other UI frameworks. Images can be stored in any folder, including alongside your content. 
+
+Store your images in the `public/` folder only if you want to avoid any processing.
+
+### Remote images
+
+You can also choose to store your images remotely, in a content management system (CMS) or digital asset management (DAM) platform. 
 
-Follow the [Assets (Experimental) Guide](/en/guides/assets/) to start using Astro's new image solution today!
+For extra protection when dealing with external sources, remote images will only be processed from [authorized image sources](#authorizing-remote-images) specified in your configuration. However, any remote images can be displayed.
 
-**Some of the advice below is incompatible with the experimental flag. Please see the Assets page if you are using `astro:assets`.**
+Astro can fetch your data remotely using APIs or display images from their full URL path. See our [CMS guides](/en/guides/cms/) for examples of integrating common services.
+
+
+## Images in `.astro` files
+
+In `.astro` files, local images must be imported into the file in order to be used. Remote and `public/` images do not require importing.
+
+Import and use Astro's built-in `<Image />` component for optimized images using `astro:assets`. Alternatively, Astro syntax supports writing an HTML `<img>` tag directly, which skips image processing. 
+
+```astro title="src/pages/blog/MyImages.astro"
+---
+import { Image } from 'astro:assets';
+import localBirdImage from ''../../images/subfolder/localBirdImage.png';
+---
+<Image src={localBirdImage} alt="A bird sitting on a nest of eggs."/>
+<Image src="/bird-public-folder.jpg" alt="A bird." width="50" height="50"/>
+<Image src="https://example.com/remote-bird.jpg" alt="A bird." width="50" height="50"/>
+
+<img src={localBirdImage.src} alt="A bird sitting on a nest of eggs.">
+<img src="/images/bird-in-public-folder.jpg" alt="A bird.">
+<img src="https://example.com/remote-bird.jpg" alt="A bird.">
+```
+
+### `<Image />` (`astro:assets`)
+
+Use the built-in `<Image />` Astro component to display optimized versions of your local images and [configured remote images](#authorizing-remote-images).
+
+Images in the `public/` folder, as well as remote images not specifically configured in your project can also be used with the Image component, but will not be processed.
+
+`<Image />` can transform a local or authorized remote image's dimensions, file type, and quality for control over your displayed image. The resulting `<img>` tag includes `alt`, `loading` and `decoding` attributes and infers image dimensions to avoid **Cumulative Layout Shift (CLS)**.
```

Comment:
Comma III - Revenge of the Sith
````suggestion
`<Image />` can transform a local or authorized remote image's dimensions, file type, and quality for control over your displayed image. The resulting `<img>` tag includes `alt`, `loading`, and `decoding` attributes and infers image dimensions to avoid **Cumulative Layout Shift (CLS)**.
````